### PR TITLE
[SYCL] Release notes for August'20 DPCPP implementation update

### PR DESCRIPTION
**Pull request by romanovvlad**
Increase major version of libsycl.so library as well

### CODE REVIEW

File: sycl/ReleaseNotes.md [hunk 1]
```diff
@@ -1,3 +1,186 @@
...
+Release notes for the commit range 75b3dc2..414c1e5
+
+## New features
+  - Implemented basic support for the [Explicit SIMD extension](./sycl/doc/extensions/ExplicitSIMD/dpcpp-explicit-simd.md)
+    for low-level GPU performance tuning [84bf234] [32bf607] [a lot of others]
+  - Implemented support for the [SYCL_INTEL_usm_address_spaces extension](https://github.com/intel/llvm/pull/1840)
+  - Implemented support for the [Use Pinned Host Memory Property extension](doc/extensions/UsePinnedMemoryProperty/UsePinnedMemoryPropery.adoc) [e5ea144][aee2d6c][396759d]
+  - Implemented aspects feature from the SYCL 2020 provisional Specification
+    [89804af]
+
```
Comment (pvchupin) [inline sycl/ReleaseNotes.md:12]: nit: double space line

File: sycl/ReleaseNotes.md [hunk 2]
```diff
@@ -1,3 +1,186 @@
...
+  - Added support for a set of standard library functions for AOT compilation
+    [2bd5dab]
+  - Allowed use of function declarators with empty parentheses [a4f2182]
+  - The fallback implementation of standard library functions is now linked to
+    the device code, only if such functions are used in kernels only [9a8864c]
+  - Added support for recursive function calls in a constexpr context [06f667a]
+  - Added a diagnostic on attempt to capture `this` as a kernel parameter
+    [1b9f026]
+  - Added [[intel::reqd_sub_group_size()]] attribute as a replacement for
+    [[cl::reqd_sub_group_size()]] which is now depricated [b2da2c8]
```
Comment (dm-vodopyanov) [inline sycl/ReleaseNotes.md:35]: deprecated

File: sycl/ReleaseNotes.md [hunk 3]
```diff
@@ -1,3 +1,186 @@
...
+    for low-level GPU performance tuning [84bf234] [32bf607] [a lot of others]
+  - Implemented support for the [SYCL_INTEL_usm_address_spaces extension](https://github.com/intel/llvm/pull/1840)
+  - Implemented support for the [Use Pinned Host Memory Property extension](doc/extensions/UsePinnedMemoryProperty/UsePinnedMemoryPropery.adoc) [e5ea144][aee2d6c][396759d]
+  - Implemented aspects feature from the SYCL 2020 provisional Specification
+    [89804af]
+
+
+## Improvements
+### SYCL Compiler
+  - [CUDA BE] Removed unnecessary memory fence in the `sycl::group::barier`
```
Comment (dm-vodopyanov) [inline sycl/ReleaseNotes.md:16]: sycl::group::bar**r**ier

File: sycl/ReleaseNotes.md [hunk 4]
```diff
@@ -1,3 +1,186 @@
...
+    implementation which should improve performance [e2fc1b8]
+  - [CUDA BE] Added support for the sycl builtins from  relational, geometric,
```
Comment (dm-vodopyanov) [inline sycl/ReleaseNotes.md:18]: * sycl -> SYCL
* remove extra space between "from" and "relational"

File: sycl/ReleaseNotes.md [hunk 5]
```diff
@@ -1,3 +1,186 @@
...
+  - [L0 BE] Added support for specialization constants [be4e641]
+  - Improved diagnostic on attempt to submit a kernel with local size which
+    doesn't math value specified in the `sycl::intel::reqd_work_group_size`
+    attribute for the kernel [03ef819]
+  - [CUDA BE] Changed active context to be persistent [296fa1a]
+  - [CUDA BE] Changed default gpu architecture for device code to `SM_50`
+    [800e452]
+  - Added a diagnostic on attempt to create a device accessor from zero-sized
+    buffer [80b2110]
+  - Changed default backend to level zero [11ef88c]
```
Comment (dm-vodopyanov) [inline sycl/ReleaseNotes.md:69]: **L**evel **Z**ero

File: sycl/ReleaseNotes.md [hunk 6]
```diff
@@ -1,3 +1,186 @@
...
+  - Improved performance of the SYCL graph cleanup [c099e47]
+  - [L0 BE] Added support for `sycl::sampler` [f3b8cdf]
+  - Added support for `TriviallyCopyable` types to the
+    `sycl::intel::sub_group::shuffle` [d3c7b20]
+  - Implemented range simplification for queue Shortcuts [4009b8b]
+  - Changed `sycl::accessor::operator[]` to return const reference when acess
```
Comment (dm-vodopyanov) [inline sycl/ReleaseNotes.md:75]: ac**c**ess

File: sycl/ReleaseNotes.md [hunk 7]
```diff
@@ -1,3 +1,186 @@
...
+    exceptions [280b93c]
+  - Fixed `sycl::buffer` constructor which takes a contiguous container to
+    enable copy back on destruction.
+  - Added support for user-defined sub-group reductions [728429a]
+  - The `sycl::backend::level0` has been renamed to  `sycl::backend::level_zero`
+    [215f591]
+  - Extended `sycl::broadcast` to support `TriviallyCopyable` types [df6d715]
+  - Implemented `get_native` and `make_*` functions for Level Zero allowing to
+    query native handles of SYCL objects and to create SYCL objects by providing
+    a native handle: platform, device, queue, program. The feature is described
```
Comment (dm-vodopyanov) [inline sycl/ReleaseNotes.md:87]: The feature is described **in** the SYCL 2020 provisional specification (?)

**Pull request merged**